Automattic/wc-calypso-bridge
# Fix duplicated "manage your subscriptions" banner

Fixes #424.

This PR prevents the core WC addons page callback from being registered rather than removing the menu item.

To test:
* Navigate to `/wp-admin/admin.php?page=wc-addons&section=helper`
* Verify there is only one "manage your subscriptions" banner
